awx/main/validators.py

```diff
@@ -161,8 +169,10 @@ def validate_certificate(data):
     Validate that data contains one or more certificates. Adds BEGIN/END lines
     if necessary.
     """
```

Comment:
This was not doing what the comment is say. Adding BEGIN/END lines when necessary. It was adding  them regardless when I ran the negative test `test_invalid_keys()`

Reply:
This validation is only currently used for SAML auth config -- where certain SAML Identity Providers (e.g. OneLogin) provide SAML metadata XML that includes the certificate data only without the begin/end markers.